news/2024/2024-06-24-4-digit-world-cup-2024-concludes.md

```diff
@@ -0,0 +1,167 @@
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/UlJ0BW0XI4M" frameborder="0" allowfullscreen></iframe>
+</div>
+
+It is time for Semifinals! Many different countries had been predicted to possibly face off against South Korea on this stage, but it was China, seeded 14th, that turned the tournament on its head. Defying expectations, China had defeated the 3rd seed Canada in the Round of 16 and the 6th seed Brazil in the Quarterfinals, proving that they are a force to be reckoned with.
+
+On the other side, the 2nd seed South Korea had advanced to the Semifinals with relative ease, losing only a few points along the way. The match between China and South Korea promised excitement, and it delivered in full.
+
+The match began with South Korea securing a win on their first pick followed by a breakpoint, setting the tone for what seemed to be another dominant performance. However, China quickly responded with a breakpoint of their own on DT3, followed by a thrilling victory on NM5 despite a mid-map disconnect from a Chinese player, winning by a mere 3,000 points. They continued their momentum with another breakpoint on NM6, securing three consecutive points and a 3–2 lead.
+
+The teams then traded breakpoints again on the next two picks. China broke the chain with a win on their DT1 pick, bringing them to match point at 5–3. South Korea fought back with a strong performance on FM1, narrowing the gap to 5–4. The final pick, NM4, saw China clinching victory with a narrow 40,000 point lead. This remarkable win knocked another top seed, South Korea, down to the losers' bracket, where they went on to face Canada. Meanwhile, China advanced to take on the United States in the Finals.
+
+### Grand Finals: ::{ flag=US }:: United States vs. ::{ flag=CA }:: Canada
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/2DlShXHRRwc" frameborder="0" allowfullscreen></iframe>
+</div>
+
+With a rematch of last year's Grand Finals, the Canadian team once again stands up to fight the United States for the top spot on the podium after a narrow 6–7 loss in 2023. Featuring the hardest mappool in the tournament series to date, this stage isn't trivially easy for anyone to perform on.
+
+Canada had a tough journey this year, dropping to the losers' bracket early after a defeat to China in the Round of 16. They managed to put together an impressive comeback though, knocking out several formidable opponents including the 3rd seed South Korea as well as China, the very team that had sent them to the losers' bracket earlier. Meanwhile, the United States maintained their dominance throughout, showing no major weaknesses on their way to the Grand Finals. The question now was: will this match be as close and thrilling as last year, or a more one-sided showing?
```

Comment:
is this just me or is China being bundled together with South Korea in the same sentence with no separation (like a comma) just awkward? No tsure if a comma would work best or just rewording this.
`[...]knocking out several formidable opponents including the 3rd seed South Korea as well as China[...]`

Reply:
would `[...]South Korea and later China[...]` go better

Reply:
yessir